### PR TITLE
Change properties key in YAML.mapping to reduce chance of conflict

### PR DESCRIPTION
**Pull request by Sija**
Followup to #5180.

### CODE REVIEW

File: src/yaml/mapping.cr [hunk 1]
```diff
@@ -216,7 +216,7 @@ module YAML
 
   # This is a convenience method to allow invoking `YAML.mapping`
   # with named arguments instead of with a hash/named-tuple literal.
-  macro mapping(**properties)
-    ::YAML.mapping({{properties}})
+  macro mapping(**_properties_)
```
Comment (oprypin) [inline src/yaml/mapping.cr]: I'm pretty sure that changing these 2 lines is all that's needed. No idea why the other PR went that far.

Comment (bew) [reply src/yaml/mapping.cr]: I don't think you need to change this macro, the splat arg name cannot be used in a call IIRC, so cannot clash

Comment (oprypin) [reply src/yaml/mapping.cr]: That too, but I guess it means this is just a bug in macro splat

Comment (bew) [reply src/yaml/mapping.cr]: Nevermind, forget what I said, I didn't know one could do https://carc.in/#/r/37a9 to mention that splat

Comment (straight-shoota) [reply src/yaml/mapping.cr]: @oprypin Both need to be renamed. Otherwise `mapping(properties: String)` would still match the other signature.

**Comment by Sija** [comment]
ping?

**Comment by Sija** [comment]
Could we get this merged after 2 weeks of silence?

**Pull request merged**